packages/cloud/tsconfig.json

```diff
@@ -22,5 +22,7 @@
     "@kbn/cloud-plugin",
     "@kbn/share-plugin",
     "@kbn/security-plugin-types-server",
+    "@kbn/security-plugin-types-public",
+    "@kbn/core",
```

Comment:
Please do not import from core's "plugin entry point" from package code. Imports from `@kbn/core/server|public` should be replaced with imports from core's corresponding domain packages instead. 

Looking at the changes in this package, it seems this was added because of this import:

```ts
import { coreMock } from '@kbn/core/public/mocks';
```

used for 

```ts
coreMock.createStart(),
```

Please use 

```ts
import { coreLifecycleMock } from '@kbn/core-lifecycle-browser-mocks';

//...

coreLifecycleMock.createCoreStart()
```

instead